app/helpers/validation_helper.rb

```diff
@@ -43,7 +43,7 @@ def error_messages_for(object)
   
   def error_messages_formatted(errors, intro = "")
     return unless errors && !errors.empty?
-    error_messages = errors.map {|msg| content_tag(:li, msg.gsub(/^(.+)\^/, '').html_safe)}.join("\n").html_safe
+    error_messages = errors.map { |msg| content_tag(:li, msg.gsub(/^(.+)\^/, '').gsub(/\^/, '').html_safe) }.join("\n").html_safe
```

Comment:
Instead of `msg.gsub(/^(.+)\^/, '').gsub(/\^/, '')`, would `msg.gsub(/^(.*)\^/, '')` work?

----

You can also remove `.gsub(/\^/, '')` from here:

https://github.com/otwcode/otwarchive/blob/030fa4607d3c59fba2cacbe4e9c99b3c49fbc1cb/app/views/challenge_signups/_signup_form.html.erb#L53

Reply:
Done and done, thank you!